### ISSUE TITLE
Software license, MIT like your other Microsoft project?

### ISSUE THREAD

**Issue by RoundSparrow** [DrJukka/MyWifiMesh#3]
MyWifimesh seems to have no license indicator.  Is it the same as https://github.com/DrJukka/BTPollingTest/blob/master/LICENSE.txt ? - Thank you

**Comment by DrJukka** [comment]
Yes in general, all stuff under my github should have the same license, unless otherwise marked.
